### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by boozook** [pd-rs/crankstart#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rtsuk** [comment]
Hi, I'm Rob. I created crank and crankstart back in 2020 to learn more about using Rust on small devices. I recently started up a new job so I'm not going to be able to contribute much right now.

**Comment by woelper** [comment]
Hi all! Just stopping by to say a big thank you. I just received my Playdate and I am having a blast hacking around with crankstart. Super fun. I had to chuckle how easy it was from making a simple hello world app to actually see it on device - many thanks for this great library.

**Reply by boozook** [reply]
Thank you so much! Your words are what give me the strength to move this project forward.
